lib/config/definitions.js

```diff
@@ -980,6 +980,19 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'nuget',
+    description: 'Configuration object for C#/Nuget',
+    stage: 'package',
+    type: 'json',
+    default: {
+      enabled: true,
```

Comment:
Answering your question, I generally set managers to `false` at first only if I think they might cause too many people to disable them. In pip's case, it's supporting fixed versions only and not ranges so I consider it a little "incomplete" and hence opt-in for now.